### PR TITLE
Added plug integration

### PR DESCRIPTION
**Pull request by Cbturtle2**
Inserts plug below top bar

### CODE REVIEW

**Review by mxmeinhold — COMMENTED** [review]
Won't this show up on every single page? I thought we were aiming specifically to avoid that.

**Comment by RamZallan** [comment]
Yeah putting it in the nav will probably show it on every page. Idk if much convo has happened on floor about this, but I'd recommend just showing it on the [`view_dir` template](https://github.com/ComputerScienceHouse/gallery/blob/develop/gallery/templates/view_dir.html), which will show it when browsing folders, but not when viewing individual files.

File: gallery/templates/nav.html [hunk 1]
```diff
@@ -61,4 +61,9 @@
             </div>
         </div>
     </nav>
+    <center>
```
Comment (RamZallan) [inline gallery/templates/nav.html:64]: Also, the `<center>` tag is from HTML4, which some browsers render, but it isn't good practice to use, as it isn't officially a part of HTML5 (the current standard). This uses bootstrap, so you can probably just apply the `align-center` class to either the link or a wrapper `<div>` to center it.

**Review by RamZallan — CHANGES_REQUESTED** [review]

**Comment by Cbturtle2** [comment]
The latest commit should fix both of these problems

File: gallery/templates/nav.html [hunk 2]
```diff
@@ -61,4 +61,11 @@
             </div>
         </div>
     </nav>
+    <div class="text-center" id="plug_integration"></div>
+    
+    <script>
+        if (window.location.pathname == '/view/dir/1'){
+        	document.getElementById("plug_integration").innerHTML = '<a href="https://plug.csh.rit.edu"><img src="https://plug.csh.rit.edu/data" alt="Advertisements By CSH Plug!"></a>';
+        }
+    </script>
```
Comment (Mstrodl) [inline gallery/templates/nav.html:70]: Please don't do this this way... Put stuff in the `view_dir` template. You can check `directory.id` in that file and only insert the plug if it's in dir 1. (see: `{% if ... %}` directives throughout that file)

**Review by Mstrodl — CHANGES_REQUESTED** [review]

File: gallery/templates/nav.html (hunk 2 above)
Comment (Cbturtle2) [reply gallery/templates/nav.html:70]: Want to talk later tonight? Still trying to get the hang of this and a little confused lol

Comment (Mstrodl) [reply gallery/templates/nav.html:70]: Sure :call_me_hand:

**Pull request closed without merging**